src/codecTools/createDefaultCodec.ts

```diff
@@ -0,0 +1,99 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as t from 'io-ts';
+
+/**
+ *  Creates a default value for an io-ts codec.
```

Comment:
could you improve docs here? unsure what this is for

Reply:
the test has examples, even reading that im not sure how this is intended to work, and in what situations this would be useful

Reply:
Improved the description. Let me know if you're still confused.